### DISCUSSION TITLE
Monitor for Lithium battery cells

### DISCUSSION THREAD

**Discussion by Hedda** [ffenix113/zigbee_home#22] [General]
https://github.com/osaether/nrf_cell_mon

**Comment by ffenix113** [comment]
Thank you Hedda,

Would you mind instead converting this to issue?

**Reply by Hedda** [reply]
Only you as the admin of this repo can move/convert discussions to issues and vice versa ;)

**Reply by ffenix113** [reply]
Got to learn something new every day 😄 

Moved to issue https://github.com/ffenix113/zigbee_home/issues/24. Thank you.
